tdnf.spec.in

```diff
@@ -158,7 +163,7 @@ ln -sfv %{name} %{buildroot}%{_bindir}/tyum
 ln -sfv %{name} %{buildroot}%{_bindir}/yum
 ln -sfv %{name} %{buildroot}%{_bindir}/tdnfj
 pushd build/python
-python3 setup.py install --skip-build --prefix=%{_prefix} --root=%{buildroot}
+python3 setup.py install --skip-build --prefix=%{_prefix} --install-lib=%{python3_sitelib} --root=%{buildroot}
```

Comment:
Do we need this python module related change?
https://github.com/vmware/tdnf/pull/442 is open from a long time now, let's remove python related stuff.

Reply:
I'll do that in another change.